hystrix-core/src/test/java/com/netflix/hystrix/HystrixObservableCommandTest.java

```diff
@@ -2717,12 +2717,7 @@ public void testBasicExecutionWorksWithoutRequestVariable() {
             fail("We received an exception => " + e.getMessage());
         }
 
-        try {
-            HystrixRequestLog.getCurrentRequest();
-            fail("Should not have a RequestLog when RequestContext not initialized");
-        } catch (IllegalStateException ise) {
-            //expected
-        }
+        assertNull(HystrixRequestLog.getCurrentRequest());
```

Comment:
I'd been hoping to avoid changing any tests but the use of this same method call in command instantiation precludes that option without more logical changes.